types/index.d.ts

```diff
@@ -342,33 +342,48 @@ declare module 'found' {
     onClick?: (event: React.SyntheticEvent<any>) => void;
   }
 
+  interface LinkInjectedProps {
```

Comment:
shouldn't this be `ComponentPropsWithRef<'a'>`?

Reply:
eh, this seems to work?

Reply:
but DRY!